datasets/us/fincen_msb/crawler.py

```diff
@@ -0,0 +1,47 @@
+from csv import DictReader
+from typing import Dict, List
+from rigour.mime.types import CSV
+
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_row(context: Context, row: Dict[str, List[str]]):
+    name = row.pop("LEGAL NAME")
+    street = row.pop("STREET ADDRESS")
+    city = row.pop("CITY")
+    state = row.pop("STATE")
+    zip_code = row.pop("ZIP")
+    country = row.pop("FOREIGN LOCATION")
+    listing_date = row.pop("RECEIVED DATE")
+    if street and listing_date:  # check to exclude the footnotes
+        entity = context.make("LegalEntity")
+        entity.id = context.make_id(name, street, listing_date, city, state)
+        entity.add("name", name)
+        entity.add("alias", row.pop("DBA NAME"))
+        entity.add("sector", row.pop("MSB ACTIVITIES"))
+        entity.add("topics", "fin")
+        h.make_address(
+            context,
+            street=street,
+            city=city,
+            state=state,
+            postal_code=zip_code,
+            country=country,
+        )
+        context.emit(entity)
```

Comment:
Always good to audit

```suggestion
        context.emit(entity)
        context.audit(row)
```